src/requests/getForecast.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable no-console */
+import axios from "axios";
+
+const getForecast = (
+  searchText,
+  setErrorMessage,
+  setSelectedDate,
+  setForecasts,
+  setLocation
+) => {
+  let endpoint = "https://mcr-codes-weather.herokuapp.com/forecast";
+
+  if (searchText) {
+    endpoint += `?city=${searchText}`;
+  }
+
```

Comment:
What would happen if the error message is present when a new request is made? This data would still be returned and passed to the setters you use here, but it will never be displayed. This is because there is still an error message stored in `errorMessage` and the conditional rendering you have set up will just display the contents of this rather than the components you want it to.

To fix this you should add `setErrorMessage("")` here to remove it.